### DISCUSSION TITLE
how i install with no source?

### DISCUSSION THREAD

**Discussion by RaulDisconziYT** [MrMeowzz/Funkin-MrMeowzz#61] [General]
^
I
I
I

**Comment by MrMeowzz** [comment]
Do you mean just the exe? You can get it in [releases.](https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/latest)

**Reply by RaulDisconziYT** [reply]
no, the folder

**Reply by MrMeowzz** [reply]
? like what folder
